src/renderers/dom/client/syntheticEvents/SyntheticEvent.js

```diff
@@ -90,17 +100,21 @@ function SyntheticEvent(dispatchConfig, targetInst, nativeEvent, nativeEventTarg
 assign(SyntheticEvent.prototype, {
 
   preventDefault: function() {
+    if (this._destructored) {
```

Comment:
As far as I know, in production, this will become an empty branch:

```js
if (this._destructored) {
}
```

This is why it seems better to wrap the whole thing in `__DEV__` rather than the other way around.